modules/conversion-center/base/widget-contact-button-base.php

```diff
@@ -469,6 +472,73 @@ private function add_style_chat_button_section(): void {
 
 		$this->end_controls_tabs();
 
+		$this->add_control(
+			'style_chat_button_box_divider',
+			[
+				'type' => Controls_Manager::DIVIDER,
+			]
+		);
+
+		$this->add_group_control(
+			Group_Control_Box_Shadow::get_type(),
+			[
+				'name' => 'style_chat_button_box_shadow',
+				'selector' => '{{WRAPPER}} .e-contact-buttons__chat-button',
+			]
+		);
+
+		$this->add_responsive_control(
+			'style_chat_button_animation',
+			[
+				'label' => esc_html__( 'Entrance Animation', 'elementor' ),
+				'type' => Controls_Manager::ANIMATION,
+				'frontend_available' => true,
+			]
+		);
+
+		$this->add_control(
+			'style_chat_button_animation_duration',
+			[
+				'label' => esc_html__( 'Animation Duration', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'default' => [
+					'size' => 800,
+				],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 3000,
+						'step' => 100,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .e-contact-buttons' => '--e-contact-button-chat-button-animation-duration: {{SIZE}}ms;',
+				],
+				'condition' => [
+					'style_chat_button_animation!' => '',
+				],
+			]
+		);
+
+		$this->add_control(
+			'style_chat_button_animation_delay',
+			[
+				'label' => esc_html__( 'Animation Delay', 'elementor' ) . ' (ms)',
+				'type' => Controls_Manager::NUMBER,
+				'default' => '',
```

Comment:
Should the default value of the delay be an empty string? and not 0?
IN the Figma it says 1500ms which doesn't make much sense

Reply:
Yeah, it should be 0

Reply:
Fixed.